site/content/docs/5.3/components/card.md

```diff
@@ -423,10 +423,11 @@ Cards include various options for customizing their backgrounds, borders, and co
 
 Set a `background-color` with contrasting foreground `color` with [our `.text-bg-{color}` helpers]({{< docsref "helpers/color-background" >}}). Previously it was required to manually pair your choice of [`.text-{color}`]({{< docsref "/utilities/colors" >}}) and [`.bg-{color}`]({{< docsref "/utilities/background" >}}) utilities for styling, which you still may use if you prefer.
 
+<!-- Boosted mod: move text-bg-{{ .name }} on card-body -->
```

Comment:
This Boosted mod should be removed because not true anymore (and the "End mod" line 440)